src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -596,9 +596,18 @@ def reload_module(module):
 
 def get_default_admin_username():
     try:
-        return getpass.getuser()
+        username = getpass.getuser()
     except KeyError:
-        return None
+        username = None
```

Comment:
When `KeyError` is raised, `username` will be `None`, which will make `usernmae.lower()` fail.

Reply:
Oh! Thank you for finding this bug!